pkg/cli/admin/nodeimage/create.go

```diff
@@ -0,0 +1,54 @@
+package nodeimage
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+func NewCreate(f kcmdutil.Factory, streams genericiooptions.IOStreams) *cobra.Command {
+	o := NewCreateOptions(streams)
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create an ISO image for booting the nodes to be added to the target cluster",
```

Comment:
```suggestion
		Short: "Experimental: Create an ISO image for booting the nodes to be added to the target cluster",
```

Convention is firstly releasing commands as `Experimental` and subsequent releases marking them as GA'd by removing the `Experimental` prefixes.

Reply:
Not sure if it applies to this specific case - where the DevPreview feature has been released in a different form - but https://issues.redhat.com/browse/OCPSTRAT-1316 it's directly a GA feature. cc @racedo

Reply:
This may tie to GA feature but these commands are released now and command itself is marked as experimental. Isn't this too strong justification for this newly implemented command is ready for GA?

Reply:
Perhaps we could mark as Experimental for now, but aim to remove that before 4.17 branches?

Reply:
This is not a blocker for me and I'm fine with your any decision. But conventionally I'd recommend to have it experimental in 4.17 and promote it to GA in 4.18.

Reply:
@ardaguclu your point looks reasonable to me since from the point of view of `oc` it's a freshly new command. Could be acceptable to mark the command as `Experimental` in 4.17 (where the feature is marked as GA) and remove it in 4.18?

Reply:
See comment https://github.com/openshift/oc/pull/1808#issuecomment-2222418180